docs/config/cluster.rst

```diff
@@ -1285,7 +1285,8 @@ Overload Protection
 -------------------
 
 Overload protection settings control how many resources operations like
-``INSERT INTO FROM QUERY`` or ``COPY`` can use.
+``INSERT INTO FROM QUERY``, ``INSERT FROM VALUES``, ``UPDATE``, ``DELETE``
```

Comment:
`INSERT FROM VALUES` is not using the overload protection to leave pressure and speed up to the client.
```suggestion
``INSERT INTO FROM QUERY``,  ``UPDATE``, ``DELETE``
```